examples/vanilla-ts-esm/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "dependencies": {
     "@mux-elements/mux-audio": "^0.2.0",
-    "@mux-elements/mux-player": "^0.1.0-alpha.3",
+    "@mux-elements/mux-player": "^0.1.0-alpha.7",
```

Comment:
This example code in my repo wasn't correctly linked until I changed this, then it was picking up my changes.

Am I doing something wrong in my local setup? Do I need to run `npm link` or something?